middleware/csrf.go

```diff
@@ -67,12 +71,13 @@ type (
 var (
 	// DefaultCSRFConfig is the default CSRF middleware config.
 	DefaultCSRFConfig = CSRFConfig{
-		Skipper:      DefaultSkipper,
-		TokenLength:  32,
-		TokenLookup:  "header:" + echo.HeaderXCSRFToken,
-		ContextKey:   "csrf",
-		CookieName:   "_csrf",
-		CookieMaxAge: 86400,
+		Skipper:        DefaultSkipper,
+		TokenLength:    32,
+		TokenLookup:    "header:" + echo.HeaderXCSRFToken,
+		ContextKey:     "csrf",
+		CookieName:     "_csrf",
+		CookieMaxAge:   86400,
+		CookieSameSite: http.SameSiteDefaultMode,
```

Comment:
I think this causes it to send a SameSite header with no value in the default `http.Cookie` case, which is probably unnecessary? Same for the `if` that sets it below. Might just default to 0?

Reply:
The SameSite will not be set to headers if it has a default or empty value. This can be seen in the `TestCSRFWithoutSameSiteMode` test.